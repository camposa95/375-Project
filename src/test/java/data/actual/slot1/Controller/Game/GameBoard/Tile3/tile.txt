TerrainType: PASTURE
DieNumber: 8
HasRobber: false
VertexIDs: [7, 8, 9, 7, 18, 19]
