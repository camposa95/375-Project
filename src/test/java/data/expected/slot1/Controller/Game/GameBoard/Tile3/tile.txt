TerrainType: FIELDS
DieNumber: 12
HasRobber: false
